### ISSUE TITLE
Have you val the method on CurveLanes dataset？

### ISSUE THREAD

**Issue by wywywy01** [sel118/LaneAF#31]

**Comment by arangesh** [comment]
No, we do not as of now. If I get time, I might add support for the CurveLanes dataset in a few weeks.

**Comment by cc19860606** [comment]
Can you share the  CurveLanes dataset? @wywywy01 
I did not find a link to CurveLanes dataset.

**Comment by arangesh** [comment]
You can find the dataset here: https://github.com/SoulmateB/CurveLanes

**Comment by zjsun7** [comment]
hello, could you please tell me your torch and torchvision version，there is some error, when i make the make.sh file?
